src/microsoft-extensions-ai/abstraction-implementations/AbstractionImplementationExamples/Streaming.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.Extensions.AI;
+
+public partial class AbstractionSamples
+{
+    public static async Task Streaming() 
+    {
+        IChatClient client = new SampleChatClient(new Uri("http://coolsite.ai"), "my-custom-model");
+
+        var response = client.CompleteStreamingAsync("What is AI?");
+
+        await foreach(var message in response)
```

Comment:
Nit: the actual type here is a StreamingChatCompletionUpdate, so consider renaming "message" to "update"